src/LocaleStrings.lua

```diff
@@ -412,423 +413,517 @@ LC_RU.store = "LOTRO Store"
+LC_RU.token.POLISHED_MARBLE_TRINKET = "Полированная мраморная безделушка"
 
 LC_EN.token.HOST_OF_WEST_SILVER = "Host of the West Silver Pieces"
 LC_DE.token.HOST_OF_WEST_SILVER = ""
 LC_FR.token.HOST_OF_WEST_SILVER = ""
+LC_RU.token.HOST_OF_WEST_SILVER = "Серебряные монеты Воинства Запада"
 
-LC_EN.token.SILVER_SIGNET_OF_THANDRIM = "Silver Signets of Thandrim"
+LC_EN.token.SILVER_SIGNET_OF_THANDRIM = "Silver Signets of the Thandrim"
 LC_DE.token.SILVER_SIGNET_OF_THANDRIM = ""
 LC_FR.token.SILVER_SIGNET_OF_THANDRIM = ""
+LC_RU.token.SILVER_SIGNET_OF_THANDRIM = "Серебряные печати тандримов"
 
-LC_EN.token.TOKEN_OF_LAKE_AND_RIVERS = "Tokens of Lake and Rivers"
+LC_EN.token.TOKEN_OF_LAKE_AND_RIVERS = "Tokens of the Lake and Rivers"
 LC_DE.token.TOKEN_OF_LAKE_AND_RIVERS = ""
 LC_FR.token.TOKEN_OF_LAKE_AND_RIVERS = ""
+LC_RU.token.TOKEN_OF_LAKE_AND_RIVERS = "Жетоны Озера и Рек"
 
 LC_EN.token.MARK_OF_LONGBEARDS = "Mark of the Longbeards"
 LC_DE.token.MARK_OF_LONGBEARDS = ""
 LC_FR.token.MARK_OF_LONGBEARDS = ""
+LC_RU.token.MARK_OF_LONGBEARDS = "Печать Длиннобородов"
 
 LC_EN.token.GULMARK = "Gúlmark"
 LC_DE.token.GULMARK = ""
 LC_FR.token.GULMARK = ""
+LC_RU.token.GULMARK = "Гулмарка"
 
 LC_EN.token.VALES_BEORNING_TOKEN = "Vales - Beorning Token"
 LC_DE.token.VALES_BEORNING_TOKEN = ""
 LC_FR.token.VALES_BEORNING_TOKEN = ""
+LC_RU.token.VALES_BEORNING_TOKEN = "Долина - жетон беорнингов"
 
 LC_EN.token.SIGIL_IMLAD_ITHIL = "Sigil of Imlad Ithil"
 LC_DE.token.SIGIL_IMLAD_ITHIL = ""
 LC_FR.token.SIGIL_IMLAD_ITHIL = ""
+LC_RU.token.SIGIL_IMLAD_ITHIL = "Печать Имлад Итиля"
 
 LC_EN.token.NORTHERN_GULMARK = "Northern Gúlmark"
 LC_DE.token.NORTHERN_GULMARK = ""
 LC_FR.token.NORTHERN_GULMARK = ""
+LC_RU.token.NORTHERN_GULMARK = "Северная гулмарка"
 
 LC_EN.token.COPPER_COIN_OF_GUNDABAD = "Copper Coins of Gundabad"
 LC_DE.token.COPPER_COIN_OF_GUNDABAD = ""
 LC_FR.token.COPPER_COIN_OF_GUNDABAD = ""
+LC_RU.token.COPPER_COIN_OF_GUNDABAD = "Медные монеты Гундабада"
 
 LC_EN.token.TOKEN_OF_KHARUM_UBNAR = "Tokens of the Kharum-ubnâr"
 LC_DE.token.TOKEN_OF_KHARUM_UBNAR = ""
 LC_FR.token.TOKEN_OF_KHARUM_UBNAR = ""
+LC_RU.token.TOKEN_OF_KHARUM_UBNAR = "Жетоны Кхарум-Убнар"
 
 LC_EN.token.LEDGER_KEEPER_MARK = "Ledger-keeper Marks"
 LC_DE.token.LEDGER_KEEPER_MARK = ""
 LC_FR.token.LEDGER_KEEPER_MARK = ""
+LC_RU.token.LEDGER_KEEPER_MARK = "Марки Хранителей книг"
 
 LC_EN.token.EAGLE_BIT = "Eagle Bits"
 LC_DE.token.EAGLE_BIT = ""
 LC_FR.token.EAGLE_BIT = ""
+LC_RU.token.EAGLE_BIT = "Орлиные монеты"
 
 LC_EN.token.ZAKAF_BESHEK = "Zakaf-beshêk"
 LC_DE.token.ZAKAF_BESHEK = ""
 LC_FR.token.ZAKAF_BESHEK = ""
+LC_RU.token.ZAKAF_BESHEK = "Закаф-Бешек"
 
 LC_EN.token.SILVER_COIN_OF_GUNDABAD = "Silver Coins of Gundabad"
 LC_DE.token.SILVER_COIN_OF_GUNDABAD = ""
 LC_FR.token.SILVER_COIN_OF_GUNDABAD = ""
+LC_RU.token.SILVER_COIN_OF_GUNDABAD = "Серебряные монеты Гундабада"
 
-LC_EN.token.SARSKILLINAN = "Sárskillinan"
+LC_EN.token.SARSKILLINAN = "Sárskilinnan"
```

Comment:
Is this the spelling in-game? The original spelling was copied from https://lotro-wiki.com/wiki/Item%3ASárskillin

Reply:
Singular: Sárskillin
Plural: Sárskilinnan
![image](https://github.com/user-attachments/assets/924cfc36-e1da-439e-a19c-d5d3184abfed)